cloud/kubernetes/bring-your-own-certs/cluster-init.yaml

```diff
@@ -16,4 +16,13 @@ spec:
           - "init"
           - "--insecure"
           - "--host=cockroachdb-0.cockroachdb"
+        volumeMounts:
+          - name: client-certs
+            mountPath: /cockroach-certs
       restartPolicy: OnFailure
+      volumes:
+        - name: client-certs
+          secret:
+            secretName: cockroachdb.client.root
```

Comment:
I think we should let people know they need to create this secret. Or am I missing something that auto-creates it?